translations/bnpchs-penyisihan-2018-pergi-liburan.html

```diff
@@ -0,0 +1,86 @@
+<!-- Title : Holiday Trip -->
+
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  The holiday season has arrived, everyone is going on a holiday trip, including Jelix Fingga. 
+  Jelix Fingga, who lives in city $$$1$$$, decided to take a trip with his family to city $$$N$$$ using a car. 
+  There are $$$M$$$ bidirectional roads that connect the cities.
+</p>
+
+<p style="text-align: justify;">
+  On each road, there are landscapes with beauty value $$$W$$$. 
+  However, watching the same landscapes will surely be boring. 
+  Therefore, the total beauty value is the bitwise operator <b>OR</b> 
+  of each road traversed by Jelix Fingga until he reaches his destination.
+</p>
+
+<p style="text-align: justify;">
+  Formally, the total beauty value is defined as the following :
+</p>
+
+<p style="text-align: justify;">
+  $$$ {\Large \textrm{Beauty Value} = W_1 \cup W_2 \cup W_3 \dots} $$$
+</p>
+
+<p style="text-align: justify;">
+  Where $$$W_i$$$ refers to the beauty value of a road that is traversed by Jelix Fingga.
+</p>
+
+<p style="text-align: justify;">
+  For this trip, Jelix Fingga wants to have a memorable experience. Therefore, he wants the total beauty value when he reaches city $$$N$$$ to be $$$X$$$. However, he doesn't know if that total beauty value can be reached or not. Because he is busy preparing for the trip, he asked you, his wonderful friend, whether it is possible to reach city $$$N$$$ with the total beauty value $$$X$$$. 
```

Comment:
```suggestion
  For this trip, Jelix Fingga wants to have a memorable experience. 
  Therefore, he wants the total beauty value when he reaches city $$$N$$$ to be $$$X$$$. 
  However, he doesn't know if that total beauty value can be reached or not. 
  Because he is busy preparing for the trip, he asked you, his wonderful friend, 
  whether it is possible to reach city $$$N$$$ with the total beauty value $$$X$$$. 
```